status/versions.go

```diff
@@ -222,51 +222,58 @@ func checkRange(low string, high string, version string) bool {
 	return ok && ok1
 }
 
-// GetMeshInfo get mesh name/version from mesh service
-func GetMeshInfo() (string, string, error) {
+// CheckVersionCompatibility check mesh name/version compatibility with kiali
+// Only log warnings one time when starting kiali up
+func CheckVersionCompatibility() {
 	istioInfo, err := istioVersion()
 	if err != nil {
-		return "", "", err
+		log.Warning(err)
 	} else if istioInfo.Name == istioProductNameUnknown {
-		return "Unknown", "", nil
+		log.Warning("Unknown Istio implementation version " + istioInfo.Version + " is not recognized, thus not supported.")
+	} else {
+		log.Infof("Mesh Name: [%v], Mesh Version: [%v]", istioInfo.Name, istioInfo.Version)
 	}
-	return istioInfo.Name, istioInfo.Version, nil
 }
 
 // istioVersion returns the current istio version information
+// add warnings when mesh version is incompatible with kiali
 func istioVersion() (*ExternalServiceInfo, error) {
 	istioConfig := config.Get().ExternalServices.Istio
 	body, code, err := httputil.HttpGet(istioConfig.UrlServiceVersion, nil, 10*time.Second, nil)
+
+	configWarnings := "failed to get mesh version, please check if url_service_version is configured correctly."
+
 	if err != nil {
-		AddWarningMessages("Failed to get mesh version, please check if url_service_version is configured correctly.")
-		Put(IsCompatible, "false")
```

Comment:
This old line 242 (as well as old lines 258 and 263) removes populating the "IsCompatible" field. Since you removed the code that populates it, you need to remove it from the status constants [here](https://github.com/kiali/kiali/blob/v1.47.0/status/status.go#L11):

```diff
diff --git a/status/status.go b/status/status.go
index 7a7131cc..742159dc 100644
--- a/status/status.go
+++ b/status/status.go
@@ -8,7 +8,6 @@ const (
        CoreVersion      = name + " core version"
        MeshName         = "Mesh name"
        MeshVersion      = "Mesh version"
-       IsCompatible     = "compatibility"
        CoreCommitHash   = name + " core commit hash"
        State            = name + " state"
        ClusterMTLS      = "Istio mTLS"
```